src/acceptance/java/io/codurance/pouch/GetResourceFeature.java

```diff
@@ -39,26 +42,26 @@ public void setUp() {
     @Test
     public void shouldGetOneResource() {
         var resourceOne = new DatabaseHelper.Resource(
-                1,
-                "2018-12-05 16:01:00.0",
+                UUID.fromString("0b6f4e90-1295-11e9-ab14-d663bd873d93"),
+                Timestamp.valueOf("2018-12-05 16:01:00.0"),
```

Comment:
Can we do the same with this? It would be better to make the references to this timestamp DRY.